src/Components/Users/users.jsx

```diff
@@ -1,44 +1,79 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
 import React from 'react';
+import * as axios from 'axios';
 
 import { I18N } from '../../constants';
-import usersArray from '../../usersArray';
 
 import style from './users.module.css';
 
-const users = (props) => {
-  if (props.users.length === 0) {
-    props.setUsers(usersArray);
+import defaultImg from '../Image/person.jpg';
+
+class Users extends React.Component {
+  componentDidMount() {
+    axios.get(`https://social-network.samuraijs.com/api/1.0/users?page=${this.props.currentPage}&count=${this.props.pageSize}`)
```

Comment:
Consider breaking down this API url into separate pieces (BASE_URL, USER_RESOURCE, etc.) and placing them into variable at the core/constants level.

Reply:
i've add